packages/paste-website/src/pages/components/display-pill-group/index.mdx

```diff
@@ -1,20 +1,47 @@
 ---
 title: Display Pill Group
 package: '@twilio-paste/display-pill-group'
-description: A Display Pill Group is a set of Pills used to visually represent a collection of entities outside of form-based UIs.
+description: A Display Pill Group is a non-editable set of Pills that represent a collection of static objects.
```

Comment:
issue: this change in description isn't showing up in the netlify preview. Not sure why, since the rest of the changes seem to show up just fine 🤔

Reply:
Ooops, forgot to put it in the body of the page. Should be fixed now.